generators/python/core_utilities/shared/http_client.py

```diff
@@ -121,18 +121,34 @@ def _should_retry(response: httpx.Response) -> bool:
     return response.status_code >= 500 or response.status_code in retryable_400s
 
 
+def _stringify_multipart_form_data(data: typing.Mapping[str, typing.Any]) -> typing.Dict[str, typing.Any]:
+    """
+    Convert all non-string/bytes form data values to strings for multipart requests.
+    httpx requires all form field values to be strings or bytes for multipart/form-data encoding.
+    """
+    result: typing.Dict[str, typing.Any] = {}
+    for key, value in data.items():
+        if value is None:
+            continue
+        if isinstance(value, (str, bytes)):
+            result[key] = value
+        else:
+            result[key] = str(value)
```

Comment:
can `value` here ever be a nested mapping? In which case I'm not sure that stringifying is sufficient

Reply:
is this really the correct behavior?